glib2/test/test-variant.rb

```diff
@@ -24,4 +24,71 @@ class TestGLibVariant < Test::Unit::TestCase
                    [variant.type, variant.value])
     end
   end
+
+  sub_test_case ".parse" do
+    test "value: true" do
```

Comment:
We don't need `value: ` prefix because this is not a keyword argument:

```suggestion
    test "true" do
```